lib/ts/recipe/recipeModule.ts

```diff
@@ -73,38 +73,32 @@ export default abstract class RecipeModule {
     };
 
     redirect = async (
```

Comment:
Le's discuss the code structure for redirect. It's a little hard to understand at the moment.